addons/account/tests/test_account_move_entry.py

```diff
@@ -805,3 +805,34 @@ def edit_tax_on_posted_moves():
         # You can remove journal items if the related journal entry is draft.
         self.test_move.button_draft()
         edit_tax_on_posted_moves()
+
+    def test_should_allow_creating_unbalanced_misc_entry_with_tax(self):
```

Comment:
Maybe the test needs a better name like `test_misc_entry_creation_behavior`.
It's a bit cryptic but as it's 2 tests in one...

Reply:
`test_misc_tax_autobalance`?

<details>
  <summary>:see_no_evil:</summary>
 <code>test_should_allow_creating_unbalanced_for_misc_entry_with_tax_by_autobalancing_or_should_block_by_raising_an_error</code>!
</details>